macros/staging/remove_string_from_numeric.sql

```diff
@@ -6,25 +6,25 @@
 
 {% macro default__remove_string_from_numeric(column_name) %}
     
-    cast(regexp_replace(cast({{ column_name }} as {{ dbt.type_string() }}), '[^0-9.]*', '') as {{ dbt.type_numeric() }})
+    cast(nullif(regexp_replace(cast({{ column_name }} as {{ dbt.type_string() }}), '[^0-9.]*', ''), '') as {{ dbt.type_numeric() }})
```

Comment:
In our internal data, one example was malformed, so this prevents a run failure.